indico/modules/categories/views.py

```diff
@@ -65,6 +65,13 @@ class WPCategoryCalendar(WPCategory):
 
     bundles = ('module_categories.calendar.js', 'module_categories.calendar.css')
 
+    def _get_breadcrumbs(self):
+        return render_breadcrumbs(
+            _('Calendar'),
+            category=self.category,
+            category_url_factory=lambda cat, management: url_for('categories.calendar', cat)
```

Comment:
what about using the default display page for the current category? (since you're already on the calendar page for that one anyway)

Reply:
The default view would point to `categories.display`, and not to `categories.calendar`. That'd go against what was described in #6129:

> The subcategories listed in the legend should be links to the calendar view of those subcategories.

Reply:
This is for the breadcrumbs. What I mean is that if you are e.g. on "Home -> Calendar" then "Home" links to the category display page. If you are in "Home -> Test -> Calendar" then "Test" links to the category display page and "Home" links to the calendar view.

Reply:
That'd alter the functionality of items that apparently look similar. A user would expect an item displayed in a collection of similar ones to behave similarly to other of the same nature, but what you propose would change the behaviour of a given item, which looks misleading to me.

Not a strong opinion though, so as you prefer.

Reply:
I would find this behavior rather confusing. Why would the breadcrumb links send me to different views? And what is the benefit with the last segment of the breadcrumb sending me to the category display view? Especially when there is a big button saying "Back to category page".

Reply:
I especially found it with in case of "Home -> Calendar". But I had the same worry that it might be a bit strange to have different logic. So with 2.5 people thinking it's weird, let's not do it